organize/models.py

```diff
@@ -104,9 +105,10 @@ def deploy(self):
 
         self.change_status_to(DEPLOYED)
 
-        # TODO: we should recognize here if we should create a new event,
-        # copy old one or copy old and change organizaers.
-        event = self.create_event()
+        if self.previous_event:
```

Comment:
Yeah, there is some work to be done here for sure. However, I'm not sure about the suggested solution though. Sometimes you can type city differently in the first edition: for example: Gorzów, Gorzów Wlk., Gorzów Wielkopolski. Once it could be entered without diacritical symbols, latter with these. Also it makes no sense for cities that actually have same name in the same country, i.e.: there are two Józefów in Poland. I know it's a kind of edge case, but we should do more thinking here and we might need info about lat/lng.

Let's create a separate ticket and address it once we actually have latlng properly implemented.